bluepyopt/ephys/stimuli.py

```diff
@@ -327,3 +328,88 @@ def __str__(self):
                 self.ramp_duration,
                 self.total_duration,
                 self.location)
+
+
+class LFPySquarePulse(Stimulus):
+
+    """Square pulse current clamp injection"""
+
+    def __init__(self,
+                 step_amplitude=None,
+                 step_delay=None,
+                 step_duration=None,
+                 total_duration=None,
+                 location=None):
+        """Constructor
+        Args:
+            step_amplitude (float): amplitude (nA)
+            step_delay (float): delay (ms)
+            step_duration (float): duration (ms)
+            total_duration (float): total duration (ms)
+            location (Location): stimulus Location
+        """
+
+        super(LFPySquarePulse, self).__init__()
+        self.step_amplitude = step_amplitude
+        self.step_delay = step_delay
+        self.step_duration = step_duration
+        self.location = location
+        self.total_duration = total_duration
+        self.iclamp = None
+
+    def instantiate(self, sim=None, icell=None, LFPyCell=None):
```

Comment:
This argument should not have uppercases in it, it's a variable, not a classname